docs/quickstart.md

```diff
@@ -0,0 +1,134 @@
+# Quickstart
+
+## Installation
+
+You can install **Cluster Experiments** via pip:
+
+```bash
+pip install cluster-experiments
+```
+
+!!! info "Python Version Support"
+    **Cluster Experiments** requires **Python 3.9 or higher**. Make sure your environment meets this requirement before proceeding with the installation.
+
+---
+
+## Usage
+
+Designing and analyzing experiments can feel overwhelming at times. After formulating a testable hypothesis,
+you're faced with a series of routine tasks. From collecting and transforming raw data to measuring the statistical significance of your experiment results and constructing confidence intervals,
+it can quickly become a repetitive and error-prone process.
+*Cluster Experiments* is here to change that. Built on top of well-known packages like `pandas`, `numpy`, `scipy` and `statsmodels`,  it automates the core steps of an experiment, streamlining your workflow, saving you time and effort, while maintaining statistical rigor.
```

Comment:
I'd make the paragraph shorter and stress what it automates: being MDE/power calculation and inference scorecards

Reply:
given the next examples, I think it's worth mentioning that you're describing the simulaiton-based power analysis, and there are other pipelines like power analysis based on normal approximation and scorecard generation

Reply:
I like the explanation style, maybe you could write a similar thing for NormalPowerAnalysis and AnalysisPlan